doc/get_started.rst

```diff
@@ -17,6 +17,8 @@ Python
 
 Below is a snippet of the full Python demo located at :code:`secure-xgboost/demo/python/basic/secure-xgboost-demo.py`. 
 
+Be sure that if your machine doesn't have an enclave available that you create an enclave in simulation mode. Look at lines 11-14 of ``secure-xgboost-demo.py`` for an example.
```

Comment:
How about rephrasing this, something along the lines of: 
"The snippet assumes that the machine supports hardware enclaves. If your machine does not have hardware enclave support, then you can simulate an enclave (for development purposes) by setting the flag `OE_ENCLAVE_FLAG_SIMULATE`, instead of `OE_ENCLAVE_FLAG_RELEASE`.